docs/pages/radiogroup.js

```diff
@@ -0,0 +1,695 @@
+        <RadioGroup.RadioButton
+          checked={favorite === 'pizza'}
+          id="favoriteSizePizzaSm"
+          label="Pizza"
+          name="favoriteFoodSm"
+          onChange={() => setFavorite( 'pizza' )}
+          value="pizza"
+          size="sm"
+        />
+        <RadioGroup.RadioButton
+          checked={favorite === 'curry'}
+          id="favoriteSizeCurrySm"
+          label="Curry"
+          name="favoriteFoodSm"
+          onChange={() => setFavorite( 'curry' )}
+          value="curry"
+          size="sm"
+        />
+        <RadioGroup.RadioButton
+          checked={favorite === 'sushi'}
+          id="favoriteSizeSushiSm"
+          label="Sushi"
+          name="favoriteFoodSm"
+          onChange={() => setFavorite( 'sushi' )}
+          value="sushi"
+          size="sm"
+        />
+      </RadioGroup>
+
+      <RadioGroup legend="What is your favorite snack?" errorMessage="Please select one" id="sizeExampleMd">
+        <RadioGroup.RadioButton
+          checked={favoriteFood === 'pizza'}
+          id="favoriteSizePizza"
+          label="Pizza"
+          name="favoriteFood"
+          onChange={() => setFavoriteFood( 'pizza' )}
+          value="pizza"
+        />
+        <RadioGroup.RadioButton
+          checked={favoriteFood === 'curry'}
+          id="favoriteSizeCurry"
+          label="Curry"
+          name="favoriteFood"
+          onChange={() => setFavoriteFood( 'curry' )}
+          value="curry"
+        />
+        <RadioGroup.RadioButton
+          checked={favoriteFood === 'sushi'}
+          id="favoriteSizeSushi"
+          label="Sushi"
+          name="favoriteFood"
+          onChange={() => setFavoriteFood( 'sushi' )}
+          value="sushi"
+        />
+      </RadioGroup>
+    </Flex>
+  );
+}
+        `}
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection
+          title="States"
+          description="Remember that disabled RadioButtons cannot be accessed by the keyboard and therefore should not contain any necessary info to complete the choice presented."
+        >
+          <MainSection.Card
+            defaultCode={`
+function RadioButtonExample() {
+  const [favorite, setFavorite] = React.useState(undefined);
+
+  return (
+      <RadioGroup legend="Which state is your favorite?" direction="row">
+        <RadioGroup.RadioButton
+          checked={false}
+          id="unchecked"
+          label="Unchecked"
+          name="stateExample"
+          onChange={() => setFavorite( 'unchecked' )}
+          value="unchecked"
+        />
+        <RadioGroup.RadioButton
+          checked={true}
+          id="checked"
+          label="Checked"
+          name="stateExample"
+          onChange={() => setFavorite( 'checked' )}
+          value="checked"
+        />
+        <RadioGroup.RadioButton
+          checked={false}
+          id="uncheckedDisabled"
+          label="Unchecked but disabled"
```

Comment:
nit: not sure if "but" is the right word for these two. "and" seems more appropriate